pkgs/http/lib/src/streamed_request.dart

```diff
@@ -32,7 +32,7 @@ class StreamedRequest extends BaseRequest {
   /// buffered.
   ///
   /// Closing this signals the end of the request.
-  EventSink<List<int>> get sink => _controller.sink;
+  StreamSink<List<int>> get sink => _controller.sink;
```

Comment:
I think that you need to update the example in the documentation for this class.